src/commands/database/mysql/user/create.tsx

```diff
@@ -0,0 +1,146 @@
+import { ExecRenderBaseCommand } from "../../../../lib/basecommands/ExecRenderBaseCommand.js";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../../rendering/process/process_flags.js";
+import { ReactNode } from "react";
+import { Flags, Args } from "@oclif/core";
+import { assertStatus } from "@mittwald/api-client-commons";
+import { Success } from "../../../../rendering/react/components/Success.js";
+import { Value } from "../../../../rendering/react/components/Value.js";
+
+type Result = {
+  mysqlUserId: string;
+};
+
+export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
+  static description = "Create a new mysql user";
+  static args = {
+    "database-id": Args.string({
+      description: "ID of the MySQL Database to create a user for",
+    }),
+  };
+  static flags = {
+    ...processFlags,
+    "database-id": Flags.string({
+      required: true,
+      description: "ID of the MySQL Database to create a user for",
+    }),
+    "access-level": Flags.string({
+      required: true,
+      description: "Access level for this mysql user",
+      options: ["readonly", "full"],
+    }),
+    description: Flags.string({
+      required: true,
+      description: "Description of the mysql user",
+    }),
+    password: Flags.string({
+      required: true,
+      description: "Password used for authentication",
+      exactlyOne: ["public-key", "password"],
```

Comment:
There is no `public-key` flag. Copy&paste error?

Reply:
sloppy, fixed.